packages/medusa/src/api/routes/admin/customer-groups/get-customer-group-customers.ts

```diff
@@ -37,21 +37,7 @@ import CustomerController from "../../../../controllers/customers"
  *     content:
  *       application/json:
  *         schema:
- *           type: object
- *           properties:
- *             customers:
- *               type: array
- *               items:
- *                  $ref: "#/components/schemas/Customer"
- *             count:
- *               type: integer
- *               description: The total number of items available
- *             offset:
- *               type: integer
- *               description: The number of items skipped before these items
- *             limit:
- *               type: integer
- *               description: The number of items per page
+ *           $ref: "#/components/schemas/AdminCustomersListRes"
```

Comment:
Note that this endpoint references a response shape from `admin/customers/` and not `admin/customer-groups/` since it returns a list of customers.